beetsplug/fetchart.py

```diff
@@ -683,9 +683,10 @@ def get(self, album, plugin, paths):
         cover_names_str = b'|'.join(cover_names)
         cover_pat = br''.join([br"(\b|_)(", cover_names_str, br")(\b|_)"])
 
-        for path in paths:
+        for _path in paths:
+            path = _path
```

Comment:
Is there a reason to do this? The following is simpler;

```Python
for path in paths:
    if not os.path.isdir(syspath(path)):
        path = os.path.dirname(path)
```

Reply:
You are right indeed.